project.js

```diff
@@ -367,28 +367,24 @@ function getSomeProjects(listOffset) {
+        // Flag invalid project data elements
+        var validProject = false;
```

Comment:
Adding a flag here and then testing the flag later improves the purpose of the code.